### PR TITLE
Add Image component & code snippets

### PR DESCRIPTION
**Pull request by VLeads**
Created Image component. 
Added code snippets to previous components.
And some minor changes done.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -104,10 +106,11 @@ <h3>Basic alerts</h3>
             <p class="code-snippet-heading">Code Snippets</p>
             <div class="code-snippet">
               <iframe
+                class="code-snippet-code"
                 src="https://carbon.now.sh/embed/nlm6Xyfb6HW1khg7oZWK"
                 style="
                   width: 100%;
-                  height: 298px;
+                  height: 260px;
```
Comment (nayyyhaa) [inline index.html:113]: you can remove Inline styling for iframes & can include them in CSS files. Otherwise, nice work 💯

Comment (VLeads) [reply index.html:113]: Thanks for the suggestion. But actually, I have given different heights to each Iframe that's why they have inline-style.

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Everything Else looks good

**Comment by VLeads** [comment]
Thanks @nayyyhaa @vishal804 for the review.

**Pull request merged**